### DISCUSSION TITLE
wildrift-gold-efficiency/5.1d/

### DISCUSSION THREAD

**Discussion by giscus** [changchiyou/wildrift-gold-efficiency#16] [Announcements]
# wildrift-gold-efficiency/5.1d/

Wild Rift Gold Efficiency of 5.1d

https://changchiyou.github.io/wildrift-gold-efficiency/5.1d/

**Comment by changchiyou** [comment]
~~The option ⚡`Unstable Passives' Stats` in filter is available only in [5.1d](https://changchiyou.github.io/wildrift-gold-efficiency/5.1d/) and later patch.~~ Now, this feature is supported in all patches.
